fbw-a380x/src/wasm/systems/a380_systems/src/air_conditioning/mod.rs

```diff
@@ -321,9 +321,15 @@ impl A380AirConditioningSystem {
                 context,
                 AcscId::Acsc1(Channel::ChannelOne),
                 cabin_zones,
-                vec![
-                    ElectricalBusType::DirectCurrent(1),
-                    ElectricalBusType::AlternatingCurrent(1),
+                [
+                    [
+                        ElectricalBusType::AlternatingCurrent(1), // 103XP
+                        ElectricalBusType::DirectCurrent(1),      // 101PP
+                    ],
+                    [
+                        ElectricalBusType::AlternatingCurrent(2),  // 202XP
+                        ElectricalBusType::DirectCurrentEssential, // 4PP
```

Comment:
I suppose these are not final yet?

Reply:
Correct, this is all gone in #8086 since the ACSC is not a component of the A380 system, so just a placeholder here really.